src/stories/Interactive/Code/CodeBlock.stories.tsx

```diff
@@ -72,28 +72,47 @@ CodeBlockInteractions.args = {
   filename: fileName,
 };
 CodeBlockInteractions.play = async ({ canvasElement }) => {
+  let clipboardData: unknown = '';
```

Comment:
Can clipboardData be typed as string?

Reply:
We could but you'd have to define it in the MockImplementation as well and the type is not really required for anything.